lib/radiator_web/controllers/admin/usersettings_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule RadiatorWeb.Admin.UserSettingsController do
+  use RadiatorWeb, :controller
+
+  alias Radiator.Auth
+
+  action_fallback RadiatorWeb.Api.FallbackController
+
+  def index(conn, _params) do
+    render(conn, "usersettings.html", changeset: Auth.Register.change_user(%Auth.User{}))
+  end
+
+  def create(conn, params) do
```

Comment:
Maybe `update`? You're not creating anything.

Reply:
I saw that the `resources` function was used in the admin routes. And there i think the create method which stands for `POST /admin/usersettings` is the best match.

Should i switch the routes to not use the `resources` function, and define the http methods directly?

Reply:
You probably need more flexibility here than the `resources` macro provides. It's ok to define custom routes and ideal http methods.